src/System.Collections/src/System/Collections/Generic/Stack.cs

```diff
@@ -239,6 +241,12 @@ public T[] ToArray()
             return objArray;
         }
 
+        private void ThrowInvalidOperationEmptyStack()
```

Comment:
Actually, I'm partial to the one I had in the other PR. Although it's less descriptive, it's more terse. I'll remove the inconsistency if this will be merged.